src/couch_replicator_scheduler.erl

```diff
@@ -411,13 +416,14 @@ consecutive_crashes(History, HealthThreshold) when is_list(History) ->
 consecutive_crashes([], _HealthThreashold, Count) ->
     Count;
 
-consecutive_crashes([{{crashed, _}, CrashT}, {started, StartT} | Rest],
+consecutive_crashes([{{crashed, _}, CrashT}, {PrevEvent, PrevT} | Rest],
```

Comment:
it would be clearer to say `{_, PrevT} = PrevEvent` and then pass `PrevEvent` whole later.